Framework/Crystal/src/IndexPeaks.cpp

```diff
@@ -412,6 +412,23 @@ std::map<std::string, std::string> IndexPeaks::validateInputs() {
     helpMsgs[Prop::PEAKSWORKSPACE] = exc.what();
   }
 
+  // get all runs which have peaksin the table
+  bool commonUB = this->getProperty(Prop::COMMONUB);
+  if (commonUB) {
+    auto &allPeaks = ws->getPeaks();
+    std::unordered_map<int, int> peaksPerRun;
+    auto it = allPeaks.begin();
+    while (peaksPerRun.size() < 2 && it != allPeaks.end()) {
+      peaksPerRun[it->getRunNumber()] = 1;
+      ++it;
+    };
+    if (peaksPerRun.size() < 2) {
+      helpMsgs[Prop::COMMONUB] =
+          "CommonUBForAll can only be True if there are peaks from more"
```

Comment:
Really minor but can we add a space at the end of the string or the message looks like `... peaks from morethan one ...`